src/main/kotlin/wu/seal/jsontokotlin/ui/AdvancedAnnotationTab.kt

```diff
@@ -15,90 +15,93 @@ import javax.swing.ScrollPaneConstants
+                    jRadioButton("Jackson", ConfigManager.targetJsonConverterLib == TargetJsonConverter.Jackson, {
                         ConfigManager.targetJsonConverterLib = TargetJsonConverter.Jackson
-                    }().addToGroup()
+                    })
 
-                    radioButton("Fastjson", ConfigManager.targetJsonConverterLib == TargetJsonConverter.FastJson) {
+                    jRadioButton("Fastjson", ConfigManager.targetJsonConverterLib == TargetJsonConverter.FastJson, {
                         ConfigManager.targetJsonConverterLib = TargetJsonConverter.FastJson
-                    }().addToGroup()
+                    })
 
-                    radioButton("MoShi(Reflect)", ConfigManager.targetJsonConverterLib == TargetJsonConverter.MoShi) {
+                    jRadioButton("MoShi (Reflect)", ConfigManager.targetJsonConverterLib == TargetJsonConverter.MoShi, {
                         ConfigManager.targetJsonConverterLib = TargetJsonConverter.MoShi
-                    }().addToGroup()
+                    })
 
-                    radioButton("MoShi (Codegen)", ConfigManager.targetJsonConverterLib == TargetJsonConverter.MoshiCodeGen) {
+                    jRadioButton("MoShi (Codegen)", ConfigManager.targetJsonConverterLib == TargetJsonConverter.MoshiCodeGen, {
                         ConfigManager.targetJsonConverterLib = TargetJsonConverter.MoshiCodeGen
-                    }().addToGroup()
-
-                    radioButton("LoganSquare", ConfigManager.targetJsonConverterLib == TargetJsonConverter.LoganSquare) {
+                    })
+                    jRadioButton("LoganSquare", ConfigManager.targetJsonConverterLib == TargetJsonConverter.LoganSquare, {
                         ConfigManager.targetJsonConverterLib = TargetJsonConverter.LoganSquare
-                    }().addToGroup()
-
-                    radioButton("kotlinx.serialization", ConfigManager.targetJsonConverterLib == TargetJsonConverter.Serializable) {
+                    })
+                    jRadioButton("kotlinx.serialization", ConfigManager.targetJsonConverterLib == TargetJsonConverter.Serializable, {
                         ConfigManager.targetJsonConverterLib = TargetJsonConverter.Serializable
-                    }().addToGroup()
-
-                    val customizeAnnotationRadioButton = radioButton("Others by customize", ConfigManager.targetJsonConverterLib == TargetJsonConverter.Custom) {
+                    })
+                    jRadioButton("Others by customize", ConfigManager.targetJsonConverterLib == TargetJsonConverter.Custom, {
                         ConfigManager.targetJsonConverterLib = TargetJsonConverter.Custom
-                    }().addToGroup()
-
-                    onRadioButtonSelectListener = {
-                        callBack(it == customizeAnnotationRadioButton)
+                    }) {
+                        addChangeListener {
+                            callBack(isSelected)
```

Comment:
It seems That the logic is not the same with before

Reply:
I think the logic is the same as before. just simpler than before.